casper/src/main/resources/AuthKey.rho

```diff
@@ -69,7 +69,7 @@ in {
   } |
   // No internal state to return.
   // Method required for updates.
-  contract @(*AuthKey, "extractState")(ret) = {
+  for (ret <- @(*AuthKey, "extractState")) {
```

Comment:
Could you explain the reason and effect of this change? A I understand,  persistent receive is replaced with linear one, so I was expecting to have some produce somewhere that puts the extracted state back.

Reply:
Because right not , `AuthKey` could be reused in update and we could bound new state with only one consume.